project-base/storefront/components/Forms/validationRules.ts

```diff
@@ -7,6 +7,10 @@ import { Schema } from 'yup';
 export const validateEmail = (t: Translate): Schema => {
     return Yup.string()
         .required(t('Please enter email'))
+        .matches(
+            /^[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/i,
```

Comment:
Well it's not very readable, but that goes without saying to all complex regexes :-D as long as it works, fine by me

Reply:
Guys on Breno did some research, I did too. So its regex approved by the community :)

Reply:
Yes on Breno we used  regex by RFC 2822 specification, as far as I know works fine :)